src/matrix_matrix/HYPRE_BuildIJMatrixFromDistributedMatrix.c

```diff
@@ -16,9 +16,9 @@
 #endif
 
 #include <HYPRE_config.h>
-
+#ifndef WIN32
```

Comment:
Why is this needed?

Reply:
Not sure, this was probably put in to get the Windows build with Visual Studio working. I can try removing it over on our side to see if it's still necessary.

Reply:
My question would be, if `general.h` is not needed on the windows side, why is it needed at all?  Let me know what you find out.  Thanks!

Reply:
I've removed the #ifndef, as it doesn't seem to be doing anything.